src/analyzer/protocol/ssl/SSL.cc

```diff
@@ -8,6 +9,73 @@
 #include "zeek/analyzer/protocol/ssl/ssl_pac.h"
 #include "zeek/analyzer/protocol/ssl/tls-handshake_pac.h"
 
+#include <arpa/inet.h>
+#include <openssl/evp.h>
+#include <openssl/kdf.h>
+
+#define MSB(a) ((a>>8)&0xff)
+#define LSB(a) (a&0xff)
+
+static void fmt_seq(uint32_t num, u_char* buf)
```

Comment:
these probably all should end up in the zeek::analyzer::ssl namespace, unless there is a big reason to not have them there.